README.md

```diff
@@ -40,6 +40,8 @@ Make sure your code works before submitting it :D
 
 **PLEASE READ THIS**
 
+LINK TO THE CODE - https://github.com/salonishah01/NeoAlgo/blob/outermost-paranthesis/Python/other/remove_outermost_parenthesis.py
```

Comment:
@salonishah01 Do not update like this. Take reference to how other programs are added.